base-types.Rmd

```diff
@@ -139,7 +139,7 @@ is.primitive(sum)
       
     * `is.matrix(x)` tests if `length(dim(x))` is 2.
     
-    * `is.array(x)` tests if `length(dim(x))` is 1 or 3+.
```

Comment:
To make it more parallel with the previous bullet, maybe ``if `length(dim(x)) > 1``

Reply:
Thanks, edited it now.